script.module.resolveurl/lib/resolveurl/plugins/reviewrate.py

```diff
@@ -22,7 +22,7 @@
 class ReviewRateResolver(ResolveGeneric):
     name = 'ReviewRate'
     domains = ['reviewrate.net']
-    pattern = r'(?://)((?:.*\.)?reviewrate\.net)/(?:embed-)?([0-9a-zA-Z]+)'
+    pattern = r'(?://)((?:.*\.)?reviewrate\.net)/(?:embed-embed-)?([0-9a-zA-Z]+)'
```

Comment:
This wont match embed urls of the format `http://reviewrate.net/embed-bxgs722nsd.html`
Is that intentional?